python/mlx/nn/layers/activations.py

```diff
@@ -222,6 +249,15 @@ class LogSigmoid(Module):
     pass
 
 
+class PReLU(Module):
+    def __init__(self, num_parameters=1):
+        super().__init__()
+        self.weight = mx.zeros(shape=[num_parameters])
```

Comment:
Let's initialize this to something more reasonable ;), maybe `0.25` following [torch](https://pytorch.org/docs/stable/generated/torch.nn.PReLU.html)?

Reply:
good catch, I missed this and was causing issues in the comparison between torch.